vmdb/app/controllers/application_controller/performance.rb

```diff
@@ -176,7 +176,11 @@ def perf_refresh_data
   def perf_set_or_fix_dates(options)
     # Get start/end dates in selected timezone
     tz = options[:time_profile_tz] || options[:tz]  # Use time profile tz or chosen tz, if no profile tz
-    s, e = @perf_record.first_and_last_capture
+    s, e = @perf_record.first_and_last_capture('hourly')
+    if s.nil?
+      s, e = @perf_record.first_and_last_capture('realtime')
+      options[:typ] = "realtime"
```

Comment:
If the current default is to show the page with "daily" regardless of what is currently being shown, then I think I agree with @simon3z here.

But, I'd like a confirmation from a UI dev.  @h-kataria, @dclarizio, @martinpovolny?